great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -149,7 +153,16 @@ def auth_headers(self) -> Dict[str, str]:
     def _get(self, key: Tuple[str, ...]) -> dict:
         ge_cloud_url = self.get_url_for_key(key=key)
         try:
-            response = requests.get(ge_cloud_url, headers=self.auth_headers)
+            params: dict
+            # if name is included in the key, add as a param
+            if len(key) > 2 and key[2]:
+                params = {"name": key[2]}
+                ge_cloud_url = ge_cloud_url.rstrip("/")
+                response = requests.get(
+                    ge_cloud_url, headers=self.auth_headers, params=params
+                )
+            else:
+                response = requests.get(ge_cloud_url, headers=self.auth_headers)
```

Comment:
```suggestion
            params: Optiona[dict] = None
            # if name is included in the key, add as a param
            if len(key) > 2 and key[2]:
                params = {"name": key[2]}
                ge_cloud_url = ge_cloud_url.rstrip("/")
                
            response = requests.get(
                ge_cloud_url, headers=self.auth_headers, params=params
            )
```
Not necessary but a small suggestion since `params` is by default a `NoneType`

Reply:
s/Optiona/Optional/

Reply:
I originally had it this way, have to change some tests to expect `params=None` but good call.